stories/topics/con-gcp-upgrade-restore-from-stack.adoc

```diff
@@ -0,0 +1,78 @@
+[id="gcp-upgrade-restore-from-stack"]
+
+= Restoring the stack from backups
+
+[NOTE]
+====
+The following procedures are optional.
+====
+
+The restore process deploys a new deployment, and restores the filestore and SQL database instance to the specified backup.
+
+[NOTE]
+====
+The restored filestore instance uses CIDR network range 192.168.244.0/29, which is different from the default network filestore network range of the backup deployment.
+====
+
+.Procedure
+The following instructions describe how to run the restore playbook.
+
+. Gather the following input variables to create an extra_vars.yaml file:
```

Comment:
The `extra_vars.yml` file contains the following input variables:

(I checked and they want to standardise on .yml files.)